mod/developers/start.php

```diff
@@ -81,33 +82,70 @@ function developers_process_settings() {
 	}
 }
 
-function developers_setup_menu() {
```

Comment:
lol... you liked your own comment :)

i added a check for all removed functions as a todo